server/src/constants.ts

```diff
@@ -45,3 +45,7 @@ export let resExt = ".res";
 export let resiExt = ".resi";
 export let cmiExt = ".cmi";
 export let startBuildAction = "Start Build";
+
+// bsconfig defaults according configuration schema (https://rescript-lang.org/docs/manual/latest/build-configuration-schema)
+export let bsconfigModuleDefault = "commonjs";
```

Comment:
Tested on the sample project in `analysis/tests`: it fails to find `lib/js/...` as it's looking into `lib/commonjs/...`.
The project's bsconfig does not speicify any of the relevant data, so relies on all the defaults.